src/elements/logo/logo.ts

```diff
@@ -23,22 +23,41 @@ export class SbbLogoElement extends SbbNegativeMixin(LitElement) {
   /** Accessibility label which will be forwarded to the SVG logo. */
   @property({ attribute: 'accessibility-label' }) public accessibilityLabel = 'Logo';
 
+  /**
+   * Display the signet without text.
+   *
+   * @internal
```

Comment:
I think it could public with good explanations. 
Please add to the logo readme that the signet only case should carefully be considered and normally not be used for public websites
What do you think @kyubisation?

Reply:
Logo reverted